python/common/org/python/types/NoneType.java

```diff
@@ -6,6 +6,167 @@
 
     NoneType() {}
 
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __pos__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary +: 'NoneType'");
+    }
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __neg__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary -: 'NoneType'");
+    }
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __invert__() {
+        throw new org.python.exceptions.TypeError("bad operand type for unary ~: 'NoneType'");
+    }
+
+
+    @org.python.Method(
+        __doc__=""
+    )
+    public org.python.Object __bool__() {
+        return new org.python.types.Bool(false);
+    }
+
+    @org.python.Method(
+        __doc__ = ""
+    )
+    public org.python.Object __ge__(org.python.Object other) {
+        if(other instanceof org.python.types.Bool) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= bool()");
+        } else if(other instanceof org.python.types.Float) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= float()");
+        } else if(other instanceof org.python.types.Int) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= int()");
+        } else if(other instanceof org.python.types.Str) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= str()");
+        } else if(other instanceof org.python.types.List) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= list()");
+        } else if(other instanceof org.python.types.Set) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= set()");
+        } else if(other instanceof org.python.types.Tuple) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= tuple()");
+        } else if(other instanceof org.python.types.Dict) {
+            throw new org.python.exceptions.TypeError("unorderable types: NoneType() >= dict()");
+        }
```

Comment:
Is there any way to reduce the repetition here? `Python.typeName(obj.getClass())` will give you the class name for *any* class, so you should be able to simplify quite a bit.